Source/ARTRealtime.m

```diff
@@ -629,12 +632,10 @@ - (ARTEventListener *)transitionSideEffects:(ARTConnectionStateChange *)stateCha
                 _connectionLostAt = [NSDate date];
                 [self.logger verbose:@"RT:%p set connection lost time; expected suspension at %@ (ttl=%f)", self, [self suspensionTime], self.connectionStateTtl];
             }
-            NSTimeInterval retryInterval = self.options.disconnectedRetryTimeout;
-            // RTN15a - retry immediately if client was connected
-            if (stateChange.previous == ARTRealtimeConnected && _shouldImmediatelyReconnect) {
-                retryInterval = 0.1;
-            }
-            [stateChange setRetryIn:retryInterval];
+            BOOL immediately = stateChange.previous == ARTRealtimeConnected && _shouldImmediatelyReconnect; // RTN15a - retry immediately if client was connected
```

Comment:
```suggestion
            const BOOL immediately = stateChange.previous == ARTRealtimeConnected && _shouldImmediatelyReconnect; // RTN15a - retry immediately if client was connected
```

Reply:
I'm going to resolve Quintin’s `const` comments here — we aren't doing it in general in the rest of the codebase and haven't been vigilant about it in PRs in general.